### PR TITLE
[Area 1] PDF migration Business Email Compromise

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-4692

### CODE REVIEW

File: content/email-security/_partials/_bec-directory-guide-compromise-list.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  list: never
+---
+
+Now that Area 1 has been authorized to access and retrieve directory information, you will need to configure the Business Email Compromise list.
+
+1. Log in to the [Area 1 dashboard](https://horizon.area1security.com/), and select **Settings** (the gear icon).
+
+2. Go to **Email Configuration** > **Enhanced Detections** > **Business Email Compromise**.
+
+    ![Access Business Email Compromise in Area 1 dashboard to star setting up this feature](/email-security/static/bec/o365/step2-business-email-compromise.png)
```
Comment (pedrosousa) [inline content/email-security/_partials/_bec-directory-guide-compromise-list.md]:
```suggestion
    ![Access Business Email Compromise in Area 1 dashboard to start setting up this feature](/email-security/static/bec/o365/step2-business-email-compromise.png)
```

File: content/email-security/_partials/_bec-directory-guide-compromise-list.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+3. Use the drop-down menu and select the directory you have created in the previous step 3.
```
Comment (pedrosousa) [inline content/email-security/_partials/_bec-directory-guide-compromise-list.md]: Nit: I'd write "**Use** the drop-down menu **to** select" or "**Open** the drop-down menu **and** select".

File: content/email-security/_partials/_bec-directory-guide-compromise-list.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+4. If the initial directory synchronization has completed, the page will refresh and list groups and users. If you do not see any information, wait a few minutes as the system completes processing the initial synchronization.
+
+    ![The screen should refresh and show a list of users and groups](/email-security/static/bec/o365/step4-business-list.png)
+
+5. Select the arrow next to a group to expand it and show its members.
+
+    ![Select the arrow to expand it and show a list of its members](/email-security/static/bec/o365/step5-show-members.png)
+
+6. To protect an entire group, select the three-dots button next to it, and then **Protect**. When you protect a group, all of its members will be automatically protected. The protection markers will turn green to indicate that protection is active.
```
Comment (pedrosousa) [inline content/email-security/_partials/_bec-directory-guide-compromise-list.md]:
```suggestion
6. To protect an entire group, select the three-dots button next to it, and then select **Protect**. When you protect a group, all of its members will be automatically protected. The protection markers will turn green to indicate that protection is active.
```
I'd repeat the "select" word.

File: content/email-security/_partials/_bec-directory-guide-compromise-list.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+    ![You can protect an entire group of users. The protection markers will turn green to show that protection is active](/email-security/static/bec/o365/step6-protect-group.png)
+
+7. You can also protect individual users. Select the three-dots button next to each user you want to protect, and then **Protect**.
```
Comment (pedrosousa) [inline content/email-security/_partials/_bec-directory-guide-compromise-list.md]:
```suggestion
7. You can also protect individual users. Select the three-dots button next to each user you want to protect, and then select **Protect**.
```

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+Cloudflare Area 1 can integrate with Google to retrieve user and group information. This can be used to enforce the Business Email Compromise configuration to prevent user impersonation.
+
+## 1. Create a service account in Google for Area 1 Directory Integration
+
+You need to authorize Cloudflare Area 1 to make connections into your Google tenant to retrieve your directory details. Cloudflare recommends that you create a service account for this purpose. This account will require the following following permissions:
+
+- View group subscriptions on your domain
+- View organization units on your domain
+- View groups on your domain
+- See info about users on your domain
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]:
```suggestion
- View group subscriptions on your domain.
- View organization units on your domain.
- View groups on your domain.
- See info about users on your domain.
```

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+weight: 2
+---
+
+# Gmail directory integration
+
+Cloudflare Area 1 can integrate with Google to retrieve user and group information. This can be used to enforce the Business Email Compromise configuration to prevent user impersonation.
+
+## 1. Create a service account in Google for Area 1 Directory Integration
+
+You need to authorize Cloudflare Area 1 to make connections into your Google tenant to retrieve your directory details. Cloudflare recommends that you create a service account for this purpose. This account will require the following following permissions:
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]: Nit: These are called **privileges** below, not permissions.

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+3. In **Admin console privileges**, select the following privileges: 
+    - _Organizational Units > Read_
+    - _Users > Read_
+    - _Directory Settings > Settings >Google Support Settings_
+    - _Directory Sync > Manage Directory Sync Settings > Read Directory Sync Settings_
+
+    ![Select only the privileges mentioned here](/email-security/static/bec/gmail/step3-console-privileges.png)
+
+4. When you specify Admin console privileges, you also grand the corresponding Admin API privileges. In any case, make sure the following privileges are selected for **Admin API privileges**:
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]:
```suggestion
4. When you specify Admin console privileges, you also grant the corresponding Admin API privileges. In any case, make sure the following privileges are selected for **Admin API privileges**:
```

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+## 2. Authorize Area 1 for Directory Access with Google
+
+1. Log in to the [Area 1 dashboard](https://horizon.area1security.com/), and select **Settings** (the gear icon).
+
+2. Go to **Directories**, and select **Add Directory** to start the authorization process.
+
+    ![Go to Directories in the dashboard of Area 1, and then select Add Directory to start the authorization process](/email-security/static/bec/gmail/step2-directories.png)
+
+3. In the Add Directory configuration panel, enter fhe following details:
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]:
```suggestion
3. In the Add Directory configuration panel, enter the following details:
```

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+    - **Directory Type**: Use the drop-down menu and select **Google**.
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]: Nit: I'd write "**Use** the drop-down menu **to** select" or "**Open** the drop-down menu **and** select".

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+    - **Directory Name**: Enter a string that represents the directory. This value will be referenced in the Business Email Compromise List configuration section. For example, `Gmail`.
+    - **Sync Frequency**: Update the value to your preference.
+
+    Select **Authorize** when you are done.
+
+    ![Add the appropriate details to the configuration panel](/email-security/static/bec/gmail/step3-directory-config-panel.png)
+
+4. The Area 1 dashboard will redirect you to a Google login page. Select or enter the appropriate account to initiate the authentication process.
+
+5. Once authenticated, the system will show a dialog box with a list of the required permissions. Check all the check-boxes, and select **Continue** to authorize the change.
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]: I believe you can write either **checkboxes** or **check boxes**, but not check-boxes.

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/o365-directory-guide.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+![A list of permissions for Area 1](/email-security/static/bec/o365/permissions.png)
+
+1. Log in to the [Area 1 dashboard](https://horizon.area1security.com/), and select **Settings** (the gear icon).
+
+2. Go to **Directories**, and select **Add Directory** to start the authorization process.
+
+    ![Go to Directories in the dashboard of Area 1, and then select Add Directory to start the authorization process](/email-security/static/bec/o365/step2-directories.png)
+
+3. In the Add Directory configuration panel, enter fhe following details:
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/o365-directory-guide.md]:
```suggestion
3. In the Add Directory configuration panel, enter the following details:
```

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/o365-directory-guide.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+    - **Directory Type**: Use the drop-down menu and select **Office 365**.
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/o365-directory-guide.md]: Nit: I'd write "**Use** the drop-down menu **to** select" or "**Open** the drop-down menu **and** select".

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/o365-directory-guide.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+    - **Directory Name**: Enter a string that represents the directory. This value will be referenced in the Business Email Compromise List configuration section. For example, `Office 365`.
+    - **Sync Frequency**: Update the value to your preference.
+
+    Select **Authorize** when you are done.
+
+    ![Add the appropriate details to the configuration panel](/email-security/static/bec/o365/step3-directory-config-panel.png)
+
+
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/o365-directory-guide.md]:
```suggestion

```

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+You need to authorize Cloudflare Area 1 to make connections into your Google tenant to retrieve your directory details. Cloudflare recommends that you create a service account for this purpose. This account will require the following following permissions:
+
+- View group subscriptions on your domain
+- View organization units on your domain
+- View groups on your domain
+- See info about users on your domain
+
+Start by creating a service account. If you already have one, you can skip this step.
+
+1. Access your [Google admin console](https://admin.google.com/), and go to **@ Account** > **Admin roles**.
```
Comment (pedrosousa) [inline content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]:
```suggestion
1. Access your [Google admin console](https://admin.google.com/), and go to **Account** > **Admin roles**.
```
`@` is just the icon. :)

File: content/email-security/_partials/_bec-directory-guide-secondary-email.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+
+2. Go to **Email Configuration** > **Enhanced Detections** > **Business Email Compromise**.
+
+3. Search for the user you want to allow an alternate email address.
+
+4. Select the three-dots button > **Edit**.
+
+    ![Select edit to add alternate email addresses to your user](/email-security/static/bec/o365/step4-edit-user.png)
+
+5. In **Secondary Emails** add the additional email addresses. Place each entry on a new line. Select **Save** to finish.
```
Comment (pedrosousa) [inline content/email-security/_partials/_bec-directory-guide-secondary-email.md]: Nit: There are three sentences in this step... Consider creating a step 6 after the image for `Select **Save** to finish.`

**Review by pedrosousa — APPROVED** [review]

File: content/email-security/_partials/_bec-directory-guide-compromise-list.md (hunk 2 above)
Comment (marciocloudflare) [reply content/email-security/_partials/_bec-directory-guide-compromise-list.md]: done

File: content/email-security/_partials/_bec-directory-guide-secondary-email.md (hunk 15 above)
Comment (marciocloudflare) [reply content/email-security/_partials/_bec-directory-guide-secondary-email.md]: done

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md (hunk 6 above)
Comment (marciocloudflare) [reply content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]: done

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md (hunk 9 above)
Comment (marciocloudflare) [reply content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]: done

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md (hunk 10 above)
Comment (marciocloudflare) [reply content/email-security/email-configuration/enhanced-detections/business-email-compromise/gworkspaces-directory-guide.md]: done

File: content/email-security/email-configuration/enhanced-detections/business-email-compromise/o365-directory-guide.md (hunk 12 above)
Comment (marciocloudflare) [reply content/email-security/email-configuration/enhanced-detections/business-email-compromise/o365-directory-guide.md]: done

**Pull request merged**